### ISSUE TITLE
It is not possible to use -fobject-code in emacs

### ISSUE THREAD

**Issue by jeremyjh** [chrisdone/intero#548]
Perhaps this argument is formatted incorrectly, but I am not able to achieve the result of applying additional settings to my `repl`, which appears to be the documented purpose of this variable. I would like to always have "-fobject-code" set in the repl in my project. Is it achievable in `intero` ? 
(setq intero-extra-ghci-options '("-fobject-code"))`

**Comment by jeremyjh** [comment]
I found the issue. https://github.com/commercialhaskell/intero/blob/c7f9fe4d3990e229f3b7e57e303e2b9dd5a010b4/elisp/intero.el#L1853

This goes back to #114 I guess. Personally I'd like to be able to make different choices. In slack it was mentioned that neovim doesn't bother with this issue. I don't have the emacs chops to figure out the right way to restore this as an option, for now I just removed that code in my local copy of the module.
